diff --git a/packages/target-browser/runtime-browser/runtime.ts b/packages/target-browser/runtime-browser/runtime.ts
index 4953f9ac3..dcdba92af 100644
--- a/packages/target-browser/runtime-browser/runtime.ts
+++ b/packages/target-browser/runtime-browser/runtime.ts
@@ -1,5 +1,4 @@
 // This needs to be injected / imported before the frontend script
-
 import {
   AutostartState,
   DcNotification,
@@ -36,7 +35,7 @@ let logJsonrpcConnection = false
 
 class BrowserTransport extends WebsocketTransport {
   constructor(private callCounterFunction: (label: string) => void) {
-    super('wss://localhost:3000/ws/dc')
+    super('wss://web.delta.domain.tld/ws/dc')
   }
 
   protected _onmessage(message: yerpc.Message): void {
@@ -83,7 +82,7 @@ class BrowserRuntime implements Runtime {
   socket: WebSocket
   private rc_config: RC_Config | null = null
   constructor() {
-    this.socket = new WebSocket('wss://localhost:3000/ws/backend')
+    this.socket = new WebSocket('wss://web.delta.domain.tld/ws/backend')
 
     this.socket.addEventListener('open', () => {
       /* ignore-console-log */
